### ISSUE TITLE
2.0.0 does not need .enable()

### ISSUE THREAD

**Issue by philipengberg** [icanzilb/EasyAnimation#55]
I just installed `2.0.0` and ran my app without doing any changes to my code.

EasyAnimation still swizzles **without** having called `EasyAnimation.enable()`.

**Comment by philipengberg** [comment]
I'm also wondering, with regard to the requests for a non-swizzling version, when I read about the need to call `EasyAnimation.enable()` I immediately cheered and thought *"That must mean that there is also a `EasyAnimation.disable()`, so that I can exactly control when to use `EasyAnimation` and when not to"*, however, looking into the code that does not seem to be the case 😞

**Comment by philipengberg** [comment]
The initial issue is invalid. Something must have gone during the CocoaPod installation. `EasyAnimation` is now not enabled by default.

However, I get this

<img width="636" alt="screen shot 2017-06-20 at 07 01 47" src="https://user-images.githubusercontent.com/1840093/27317407-76acbce2-5586-11e7-80ba-e80c2fcbee9e.png">

**Comment by icanzilb** [comment]
This is true, when installed via CP that method won't be public, I'll push a fix asap

**Comment by icanzilb** [comment]
@philipengberg Can you try with https://github.com/icanzilb/EasyAnimation/releases/tag/2.0.1 ? It's on CocoaPods

**Comment by vburojevic** [comment]
Seems that calling `EasyAnimation.enable()` after enabling it should re-swizzle the methods and disable it, but that's not the case. Any ideas @icanzilb? I'd like to make a PR to fix this if possible, and add `disable` function.

**Comment by vburojevic** [comment]
Ignore, my bad. UIViewController wasn't deallocated properly so code in viewWillDissapear didn't get called. :)

**Comment by vburojevic** [comment]
Nope, still had the issue because `swizzle` was a static block property and it could be called only once.

I opened a PR for this: #59
